pokemongo_bot/__init__.py

```diff
@@ -210,22 +225,19 @@ def work_on_cell(self, cell, position):
                          if 'latitude' in fort and 'type' in fort]
                 gyms = [gym for gym in cell['forts'] if 'gym_points' in gym]
 
+                # Remove stops that are still on timeout
+                forts = filter(lambda x: x["id"] not in self.fort_timeouts, forts)
+
                 # Sort all by distance from current pos- eventually this should
                 # build graph & A* it
                 forts.sort(key=lambda x: distance(self.position[
                            0], self.position[1], x['latitude'], x['longitude']))
 
-                for fort in forts:
-                    worker = MoveToFortWorker(fort, self)
-                    worker.work()
-
-                    worker = SeenFortWorker(fort, self)
-                    hack_chain = worker.work()
-                    if hack_chain > 10:
```

Comment:
It no longer goes through all the forts. It returns control back out to the main loop. Without this the bot wouldn't consider any new paths and just go through every fort in order of distance from the initial point.

This way it goes to the nearest, then re-assess were the nearest fort is.